ouroboros-consensus-mock/src/Ouroboros/Consensus/Mock/Node.hs

```diff
@@ -61,6 +63,7 @@ instance ( LedgerSupportsProtocol      (SimpleBlock SimpleMockCrypto ext)
          , Show (ForgeStateInfo        (SimpleBlock SimpleMockCrypto ext))
          , Show (ForgeStateUpdateError (SimpleBlock SimpleMockCrypto ext))
          , Serialise ext
+         , SerialiseNodeToClient (SimpleBlock SimpleMockCrypto ext) (LedgerConfig (SimpleBlock SimpleMockCrypto ext))
```

Comment:
This is instance is declared in Ouroboros.Consensus.Mock.Ledger.Block`, no?

Reply:
Indeed, I can replace this with `Serialise (MockLedgerConfig SimpleMockCrypto ext)`